README.md

```diff
@@ -22,6 +22,34 @@ mkdir -p models
 wget -O models/md_v5a.0.0.pt https://github.com/microsoft/CameraTraps/releases/download/v5.0/md_v5a.0.0.pt
 ```
 
+## Running the detector on a stream of images
```

Comment:
Looks good to me. Can't claim to understand all of it but comments look clear and if you've tested it I'm happy